openapi/components/schemas/users/update-user-payload.json

```diff
@@ -0,0 +1,115 @@
+{
+  "UpdateUserRequestModel": {
+    "type": "object",
+    "properties": {
+      "phoneNumbers": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "pattern": "^[+]?[0-9]{1,3}?[0-9]{10}$"
+        }
+      },
+      "name": {
+        "type": "string"
+      },
+      "bloodGroup": {
+        "type": "string",
+        "enum": [
```

Comment:
This will be necessary in a many places. Please keep it in a common place.

Reply:
Updated in 7510b59ea5622893a67d478c91caa3b141d55eea